dvc/command/freeze.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import logging
+
+from dvc.command.base import CmdBase, append_doc_link
+from dvc.exceptions import DvcException
+
+logger = logging.getLogger(__name__)
+
+
+class CmdFreezeBase(CmdBase):
+    def _run(self, unfreeze):
+        for target in self.args.targets:
+            try:
+                self.repo.freeze(target, unfreeze=unfreeze)
+            except DvcException:
+                logger.exception(
+                    "failed to {}freeze '{}'".format(
+                        "un" if unfreeze else "", target
+                    )
+                )
+
+                return 1
+        return 0
+
+
+class CmdFreeze(CmdFreezeBase):
+    def run(self):
+        return self._run(False)
+
+
+class CmdUnfreeze(CmdFreezeBase):
+    def run(self):
+        return self._run(True)
+
+
+def add_parser(subparsers, parent_parser):
+    FREEZE_HELP = "Freeze DVC-files."
+    freeze_parser = subparsers.add_parser(
+        "freeze",
+        parents=[parent_parser],
+        description=append_doc_link(FREEZE_HELP, "freeze"),
+        help=FREEZE_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    freeze_parser.add_argument(
+        "targets", nargs="+", help="DVC-files to freeze."
+    )
+    freeze_parser.set_defaults(func=CmdFreeze)
+
+    UNFREEZE_HELP = "Unfreeze DVC-files."
+    unfreeze_parser = subparsers.add_parser(
+        "unfreeze",
```

Comment:
Thinking about making it `thaw` or getting rid of it in favor of `freeze -u` :thinking: